app/javascript/components/organizations-dropdown.vue

```diff
@@ -74,8 +74,12 @@ export default {
       }
       const mapUserToDefaultTeam = localStorage.mapUserToDefaultTeam ?
         JSON.parse(localStorage.mapUserToDefaultTeam) : {};
-      const defaultTeam = this.organizationHasDefaultTeam(organization) ?
+      let defaultTeam = this.organizationHasDefaultTeam(organization) ?
         this.teams.filter(team => team.id === mapUserToDefaultTeam[this.githubLogin])[0] : organizationTeams[0];
+      const personalDefaultTeamId = mapUserToDefaultTeam[this.githubLogin] ?
+        mapUserToDefaultTeam[this.githubLogin] : 0;
+      defaultTeam = personalDefaultTeamId ?
+        this.teams.filter(team => team.id === personalDefaultTeamId)[0] : defaultTeam;
```

Comment:
Creo que para que sea más clara esta parte, lo haría con un if/else if/else. `defaultTeam` en verdad tiene tres valores que varían si:
- Hay un `mapUserToDefaultTeam[this.githubLogin]`
- `this.organizationHasDefaultTeam(organization)`
- else

Reply:
ese verdad, hecho!